calculator/calculator.py

```diff
@@ -3,12 +3,17 @@
 Date: 09.09.2022
 Email: ravisankar@dummygmail.com
 """
+import math
+int sin,cos,tan
 
 def initilization():
 	print("Initiating Calculator ...")
 
-	
-
+def getTrigonometricValues(int t):
```

Comment:
* Add Doc string to the function
* Give name for the function in more pythonic way